lib/progressbar.rb

```diff
@@ -28,11 +28,15 @@ def output(num)
   end
 
   def clear
-    return if $debug
+    return if silence?
     STDOUT.print " " * 79 + "\r"
   end
 
   def calc_ratio(num)
     num / @max
   end
+
+  def silence?
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)